### ISSUE TITLE
QUESTION: is docker containers supported?

### ISSUE THREAD

**Issue by MrT-tak** [FKLC/WhatsAppToDiscord#205]
Hi,
I’m new this and docker, im wondering if I can contain wa2dc.
 Thanks,
Me

**Comment by FKLC** [comment]
Yes you can, in fact we already have it https://hub.docker.com/repository/docker/fklc/wa2dc/general

**Comment by MrT-tak** [comment]
Tysm, lgtm! Will see!
